src/main/java/build/buildfarm/rpms/server/buildfarm-server.spec

```diff
@@ -0,0 +1,58 @@
+Name:           buildfarm-server
+Version:        0.1
+Release:        0
+Summary:        Bazel Buildfarm Server
+
+License:        Apache License, v2.0
+
+Requires:       java-11-openjdk
```

Comment:
`Requires` means dependencies needed to run?
In which case maybe we only need the `jre`- not the jdk

Reply:
java-11-openjdk installs only JRE and now I have updated to java-11-openjdk-headless which is minimal java runtime